### ISSUE TITLE
Poe trade search differs from poe sniper search.

### ISSUE THREAD

**Issue by jstevens45** [licoffe/POE-sniper#78]
When setting up a filter for any ring with 50-100 life and 60-70 total elemental resistance I'm getting no hits from the tool according to what it's gathering from poetrade. When I click the poetrade link it does indeed show no listings. I then click show search from which then shows I'm searching for any item with 50-100 life explicit and pseudo 60-70 total elemental resists, which is almost correct. If I then press search again I get results. Why is the search name on poetrade different than the one I get when I click "Show search form" and then "Search" without changing anything? I think something is broken with the filter and poetrade interaction.

![screen shot 2017-11-30 at 1 59 04 pm](https://user-images.githubusercontent.com/7586658/33457501-86201940-d5d7-11e7-84cd-bab083a8fd30.png)
![screen shot 2017-11-30 at 1 59 24 pm](https://user-images.githubusercontent.com/7586658/33457502-86386ed2-d5d7-11e7-8c26-a667aa63c282.png)
Above is your tool's poe.trade search key, which generated zero results.

After clicking show search form.
![screen shot 2017-11-30 at 1 59 36 pm](https://user-images.githubusercontent.com/7586658/33457503-865294ec-d5d7-11e7-919b-dd9282ba31d1.png)

Clicking search without editing anything. Show's tons of results, but isn't limited to rings as the search params suggest above.
![screen shot 2017-11-30 at 1 59 49 pm](https://user-images.githubusercontent.com/7586658/33457504-86687b40-d5d7-11e7-937f-a810cdd6ce08.png)

**Comment by licoffe** [comment]
Yeah, somehow it's different because the item type option is not selected properly in the poe.trade form.
